sf-stream/src/convert.rs

```diff
@@ -0,0 +1,481 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::protos::extractor;
+use aptos_api_types::{
+    Event, GenesisPayload, HashValue, MoveModuleId, ScriptPayload, Transaction, TransactionInfo,
+    TransactionPayload, WriteSet, WriteSetChange,
+};
+use std::str::FromStr;
+
+pub fn convert_move_module_id(move_module_id: &MoveModuleId) -> extractor::MoveModuleId {
+    extractor::MoveModuleId {
+        address: move_module_id.address.to_string(),
+        name: move_module_id.name.to_string(),
+        special_fields: Default::default(),
```

Comment:
Special fields?  What are those?

Reply:
this is a default field for protobuf generated via rust; unsure what they are used for, we leave them all `Default`